phpBB/language/en/acp/board.php

```diff
@@ -44,7 +44,7 @@
 	'BOARD_STYLE'					=> 'Board style',
 	'CUSTOM_DATEFORMAT'				=> 'Custom…',
 	'DEFAULT_DATE_FORMAT'			=> 'Date format',
-	'DEFAULT_DATE_FORMAT_EXPLAIN'	=> 'The date format is the same as the PHP <code>date</code> function.',
+	'DEFAULT_DATE_FORMAT_EXPLAIN'	=> 'The date format is the same as the PHP <code><a href="https://secure.php.net/manual/function.date.php" target="_blank" rel="noreferrer noopener">date()</a></code> function.',
```

Comment:
I would prefer to not use "target=_blank" at all. User should be able to use there browser properly...CRLT+click or something.

Reply:
Well, it has its own logic though.

Reply:
The link to Jabber and phpBB in the ACP do not have the `target` attribute either.
I think it's best to omit and leave it to the user, on how to open it.
It's within the ACP settings pages, so no posts are losts.

Reply:
https://jakearchibald.com/2016/performance-benefits-of-rel-noopener/

https://mathiasbynens.github.io/rel-noopener/